Update Caching Service doc for 2.18.4 (#4879)
zowe/docs-site: versioned_docs/version-v2.18.x/extend/extend-apiml/api-mediation-infinispan.md

 
* **`components.caching-service.storage.infinispan.initialHosts`**

  This property specifies the list of cluster nodes (members). List of all the members separated by a comma. 
  The format is `${haInstance.hostname}[${components.caching-service.storage.infinispan.jgroups.port}]`.

* **`components.caching-service.storage.infinispan.persistence.dataLocation`**

  The path where the service keeps its data files for the Infinispan Soft-Index Cache Store.
  The default value is `data`. If you run the Caching Service in HA and the instances use the same filesystem, you have to specify a different value of the data property for each instance. For more information, see the [Soft-Index File Store](https://infinispan.org/blog/2014/10/31/soft-index-file-store).

>   :::note

Reviewer: see the comment below regarding the similar sentence

  The value is not used since version 2.18.4. If you migrate to this or later version leave this value in the configuration to be the storage migrated well. 

* **`components.caching-service.storage.infinispan.persistence.indexLocation`**

  The path where the service keeps its index data for the Infinispan Soft-Index Cache Store.
  The default value is `index`. If you run the Caching Service in HA and the instances use the same filesystem, you have to specify a different value of the index property for each instance. For more information, see the [Soft-Index File Store](https://infinispan.org/blog/2014/10/31/soft-index-file-store).
  
  :::note

>   The value is not used since version 2.18.4. If you migrate to this or later version leave this value in the configuration to be the storage migrated well.

Reviewer: ```suggestion
  The value is not used since version 2.18.4. If you migrate to this or a later version, leaving this configuration will result in a data migration.
```
Maybe it needs more context, explaining how the migration part works
